lms/static/js/instructor_dashboard/data_download.js

```diff
@@ -118,7 +118,30 @@
             this.instructor_tasks = new (PendingInstructorTasks())(this.$section);
             this.clear_display();
             this.$list_anon_btn.click(function() {
-                location.href = dataDownloadObj.$list_anon_btn.data('endpoint');
+                var url = dataDownloadObj.$list_anon_btn.data('endpoint');
+                var errorMessage = gettext('Error generating anonymous IDs. Please try again.');
+                return $.ajax({
+                    type: 'POST',
+                    dataType: 'json',
+                    url: url,
+                    error: function(error) {
+                        if (error.responseText) {
```

Comment:
This is not showing an error on the frontend.

Reply:
Was not showing for rate_limit response. After discussion we are getting rid of rate-limiting on this task.